README.md

```diff
@@ -1144,6 +1145,7 @@ See https://staticsitegenerators.net and https://www.staticgen.com
   * [Lstu](https://github.com/ldidry/lstu) - _Let's SHorten That Url_ - Lightweight URL shortener. `WTFPL` `Perl`
   * [Nimbus](https://github.com/ethanal/nimbus) - A URL shortener and file sharer with a drag-and-drop OS X menu bar client and web interface. `MIT` `Python`
   * [Polr](https://project.polr.me/) - A modern, minimalist, modular, and lightweight URL shortener. ([Source Code](https://github.com/Cydrobolt/polr)) `GPLv2` `PHP`
+  * [shorturl](https://github.com/prologic/shorturl) - shorturl is a simple url shortener with very tiny urls. ([Demo](https://url.mills.io) [Source Code](https://github.com/prologic/shorturl/)) `MIT` `Go`
```

Comment:
* s/url/URL/
 * s/urls/URLs/
 * Please remove the source code link as it is the same as the main link.